app/views/pages/privacy.html.haml

```diff
@@ -0,0 +1,30 @@
+- content_for :page_title do
+  Privacy Policy
+
+%h1.dividing.header Privacy Policy
+%p
+  Privacy takes the front stage in transientBug. That being said, don't store truly sensitive information on transientBug.
+
+%p
+  All user data is stored on transientBug servers, and the official web extensions only communicate with the transientBug API over an HTTPS connection.
+
+%p
+  The following data is sent by the official web extension when its page action is clicked: The active tabs Title and URL. That's it.
+
+%h2.dividing.header Backups and Account deletion
+%p
+  transientBug does backup it's database to help prevent catastrophic disasters (And because I'd be really sad if my own site lost my own bookmarks). At the moment these backups are manual and happen once a week or so simply because the traffic to the site is so low. These backups are stored in a secured, private Digital Ocean Space and deleted after the next backup takes place. I am looking into setting up an automatic backup system that will keep the last 7 days of backups, taken twice a day (UTC noon and midnight). Anything older than 7 days will automatically be destroyed by this system as well.
```

Comment:
> transientBug does backup it's database to help prevent catastrophic disasters (And because I'd be really sad if my own site lost my own bookmarks). At the moment these backups are manual and happen once a week or so simply because the traffic to the site is so low. These backups are stored in a secured, private Digital Ocean Space and deleted after the next backup takes place. I am looking into setting up an automatic backup system that will keep the last 7 days of backups, taken twice a day (UTC noon and midnight). Anything older than 7 days will automatically be destroyed by this system as well.

- ~help prevent catastrophic disasters~ backing up doesn't prevent disasters, it helps recover when they happen. `aid in recovery should the worst happen`
- ~And~ because ...
- ~At the moment these backups are manual and happen once a week or so simply because the traffic to the site is so low.~ This is the kind of phrase that gets suffers from bitrot, and it doesn't add much. It can be left out.
- ~These backups are stored~ `Backups are stored`
- ~I am looking into setting up an automatic backup system that will keep the last 7 days of backups, taken twice a day (UTC noon and midnight). Anything older than 7 days will automatically be destroyed by this system as well.~ This is good planning, and should probably have a ticket or a trello card, but doesn't need to be here. Instead, how about: `backups are only guaranteed up to a week old, after which they may be deleted.`

You might also add: `Backups will not be used to recover deleted accounts or bookmarks.`